src/zcash/address/unified.cpp

```diff
@@ -12,9 +12,27 @@ using namespace libzcash;
 // Unified Keys
 //
 
-std::optional<std::pair<ZcashdUnifiedSpendingKey, HDKeyPath>> ZcashdUnifiedSpendingKey::ForAccount(const HDSeed& seed, uint32_t bip44CoinType, AccountId accountId) {
+bool libzcash::HasShielded(const std::set<ReceiverType>& receiverTypes) {
+    auto has_shielded = [](ReceiverType r) {
+        // TODO: update this as support for new shielded protocols is added.
+        return r == ReceiverType::Sapling;
```

Comment:
`ReceiverType::Orchard` exists and is exposed in the API in various places but isn't recognized as a shielded protocol here. Is there a reason for that?

Reply:
Mostly because originally I was trying to add Orchard directly without first adding UA support, the two concerns got tangled up, and the presence of the Orchard receiver type is an artifact of the way the previous PR got teased apart. I can remove the Orchard receiver type now that the purpose of this branch is just adding UAs.